src/aggregate.d

```diff
@@ -36,9 +36,9 @@ import ddmd.visitor;
 
 enum Sizeok : int
 {
-    SIZEOKnone, // size of aggregate is not computed yet
-    SIZEOKdone, // size of aggregate is set correctly
-    SIZEOKfwd, // error in computing size of aggregate
+    SIZEOKnone,             // size of aggregate is not computed yet
```

Comment:
I hate to read scrumbled code during maintenance. Reading human unfriendly code wastes MY time.